ocs_ci/helpers/helpers.py

```
@@ -6962,13 +6962,32 @@ def find_cephblockpoolradosnamespace(storageclient_uid=None):
+    with config.RunWithProviderConfigContextIfAvailable():
+        cephblockpool_rns_names = [
+            cephbprns_data["metadata"]["name"]
+            for cephbprns_data in ocp.OCP(
+                kind=constants.CEPHBLOCKPOOLRADOSNS,
+                namespace=config.ENV_DATA["cluster_namespace"],
+            ).get()["items"]
+        ]
+    if storageconsumer == constants.INTERNAL_STORAGE_CONSUMER_NAME:
+        cephbpradosns = list(
+            filter(lambda x: "-builtin-implicit" in x, cephblockpool_rns_names)
+        )[0]
```

Ruff (0.15.12)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)